locales/en-US/editor.properties

```diff
@@ -136,6 +136,10 @@ DND_SUCCESS_UNZIP=Successfully unzipped <b>{0}</b>.
 # {0} will be replaced by a tar filename
 DND_SUCCESS_UNTAR=Successfully untarred <b>{0}</b>.
 
+# Image Viewer
+
+IMAGE_DIMENSIONS={0} (width) x {1} (height) pixels
```

Comment:
That’s really a nit here too, but you could use the proper multiplication sign `×` instead of `x`